### ISSUE TITLE
Not Getting Code Snippet for "Python.Referenced" capability in violation incidents 

### ISSUE THREAD

**Issue by sravsravan-deloitte** [konveyor/analyzer-lsp#506]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

v0.3.0

### Priority

Critical

### Current Behavior

Example app : https://github.com/konveyor/analyzer-lsp/tree/main/examples/python

Example Rule :

- message: python sample rule 001
  ruleID: python-sample-rule-001
  description: "Discover testing"
  tag: ["python"]
  labels:
  - konveyor.io/source=legacy
  - konveyor.io/target=python
  when:
    python.referenced:
      location: METHOD_CALL
      pattern: "hello_world"

Associated Output:

- name: konveyor-analysis
  tags:
  - python
  violations:
    python-sample-rule-001:
      description: Discover testing
      category: potential
      labels:
      - konveyor.io/source=legacy
      - konveyor.io/target=python
      incidents:
      - uri: file:///analyzer-lsp/examples/python/file_a.py
        message: python sample rule 001
        lineNumber: 2
        variables:
          file: file:///analyzer-lsp/examples/python/file_a.py

### Expected Behavior

Code snippet should present in output under incidents block .

### How Reproducible

Always (Default)

### Environment

```markdown
Default
```

**Comment by pranavgaikwad** [comment]
@sravsravan-deloitte Thank you for filing the issue. we will take a look into this.

**Comment by pranavgaikwad** [comment]
@sravsravan-deloitte This should be fixed via #522
